pkg/services/ngalert/api/tooling/definitions/prom.go

```diff
@@ -172,9 +172,9 @@ type Rule struct {
 	Health    string `json:"health"`
 	LastError string `json:"lastError,omitempty"`
 	// required: true
-	Type           v1.RuleType `json:"type"`
-	LastEvaluation time.Time   `json:"lastEvaluation"`
-	EvaluationTime float64     `json:"evaluationTime"`
+	Type           string    `json:"type"`
```

Comment:
I saw no reason to continue using upstream's type definition for this field.

It is literally just an alias for a string, Swagger does not recognize it as an enum:
```
"RuleType": {
   "title": "RuleType models the type of a rule.",
   "type": "string"
  },
```
Meanwhile it just adds complexity to the type system and extra type conversions that add clutter to code. Also, maybe Grafana will want to become a superset of prometheus in the future here anyway.